de-lib/src/main/java/org/iplantc/de/diskResource/client/views/toolbar/DiskResourceViewToolbarImpl.java

```diff
@@ -495,7 +498,13 @@ void onMoveToTrashClicked(SelectionEvent<Item> event) {
 
     @UiHandler("newFolderMi")
     void onNewFolderClicked(SelectionEvent<Item> event) {
-        presenter.onCreateNewFolderSelected(selectedFolder);
+        if(selectedFolder == null) {
+            Folder parent = drAbFactory.folder().as();
+            parent.setPath(UserInfo.getInstance().getHomePath());
+            presenter.onCreateNewFolderSelected(parent);
```

Comment:
If you're feeling ambitious, we could probably refactor this to using events instead of having a reference to the presenter 😝 Not needed for this PR though.